lib/erlef_web/templates/blog/index.html.eex

```diff
@@ -0,0 +1,71 @@
+<div class="container mt-2">
+  <% post1 = Enum.at(@posts, 0) %>
+  <%= unless is_nil(post1) do %> 
```

Comment:
These cards should be put in a partial or helper, but it's not clear yet whether we actually want to keep them.